src/PythonClient/src/quixstreams/helpers/exceptionconverter.py

```diff
@@ -0,0 +1,18 @@
+from ..exceptions.quixapiexception import QuixApiException
+from ..native.Python.InteropHelpers.InteropUtils import InteropException
+
+
+class ExceptionConverter:
+
+    @staticmethod
+    def raise_from_interop(exception: InteropException):
+        if exception.exc_type == "QuixStreams.Streaming.QuixApi.QuixApiException":
+            raise QuixApiException(exception.message)
+
+        if exception.exc_type == "System.NotImplementedException":
+            raise NotImplementedError(exception.message + "\n" + exception.exc_stack)
+
+        if exception.exc_type == "System.Exception":
```

Comment:
same recommendation here, typeof(System.Exception).FullName

Reply:
This is in the non-native generated codebase, I don't have access to this type here. Only way  Icould do it is expose the native exception's type and reflect the name out of it, which is massively overkill for this purpose.